releases/solana/2025-05-GMP-v1.0.0.md

```diff
@@ -158,13 +158,13 @@ GOVERNANCE_PROGRAM_PATH="solana-axelar/target/deploy/axelar_solana_governance.so
 MULTICALL_PROGRAM_KEYPAIR_PATH=<path/to/multicall_program_keypair.json>
 MULTICALL_PROGRAM_PATH="solana-axelar/target/deploy/axelar_solana_multicall.so"
 
-UPGRADE_AUTHORITY_PUBKEY=<base58 pubkey to set as upgrade authority>
+UPGRADE_AUTHORITY_KEYPAIR_PATH=<path/to/upgrade_authority_keypair.json>
```

Comment:
Are you sure about this change? It should work with the pubkey, otherwise we won't be able to set the authority to a multisig contract on mainnet :thinking: